src/data/datasets.js

```diff
@@ -6,10 +6,15 @@ const datasets = [
     origin: 'https://test-data-explorer.appspot.com'
   },
   {
-    name: 'AMP PD - 2019_v1beta_0220',
-    origin: 'https://amp-pd-data-explorer.appspot.com',
+    name: 'AMP PD - 2019_v1release_1015',
+    origin: 'https://amp-pd-data-explorers.appspot.com',
```

Comment:
Please use the public URLs if you can:

- data-explorer.amp-pd.org
- clinical-data-explorer.amp-pd.org

Otherwise LGTM.

Thanks!